### PR TITLE
Reduced requirements

### PR DESCRIPTION
**Pull request by peach280**
# Pull Request

I have scanned through the repository and have commented out all the extra requirements. Please review and then I will be sure and remove all the extra requirements.

## Checklist:

- [X] My code follows [OCF's coding style guidelines](https://github.com/openclimatefix/.github/blob/main/coding_style.md)
- [X] I have performed a self-review of my own code
- [X] I have made corresponding changes to the documentation
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] I have checked my code and corrected any misspellings

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,120 +1,120 @@
...
-annotated-types==0.6.0
-appdirs==1.4.4
-asciitree==0.3.3
-attrs==23.2.0
-blosc2==2.5.1
-bokeh==3.3.4
-botocore==1.34.51
-Brotli==1.1.0
-cached-property==1.5.2
+# affine==2.4.0
```
Comment (peterdudfield) [inline requirements.txt:1]: could you remove them, not comment them out. They will be in the git history

File: satip/app.py [hunk 2]
```diff
@@ -65,7 +65,7 @@
 )
 @click.option(
     "--history",
-    default="60 minutes",
+    default="120 minutes",
```
Comment (peterdudfield) [inline satip/app.py:68]: could you put this back to 60 mins as deafult

File: requirements.txt (hunk 1 above)
Comment (peach280) [reply requirements.txt]: I have removed extra requirements now

File: satip/app.py (hunk 2 above)
Comment (peach280) [reply satip/app.py]: I have put this back to 60 mins

**Comment by peterdudfield** [comment]
This looks good, how do you know you havent missed a requirement?

**Review by peterdudfield — APPROVED** [review]
Lets merge this to `dev`, just to check the tests run e.t.c

**Comment by peterdudfield** [comment]
Happy to merge @peach280 ?

**Pull request merged**